portfolio/src/main/webapp/script.js

```diff
@@ -45,12 +45,36 @@ content.appendChild(
     link: "https://vedantroy.github.io/",
   })
 );
-fetch("/data")
-  .then((resp) => resp.json())
-  .then((commentsArray) => {
-    for (const comment of commentsArray) {
-      const p = document.createElement("p");
-      p.textContent = comment;
-      document.body.append(p);
-    }
+
+let maxComments = 3;
+const form = document.getElementById("max-comments-form");
+form.addEventListener("submit", (e) => {
+  e.preventDefault();
+  const data = new FormData(form);
+  const newMaxComments = parseInt(data.get("max-comments"));
+  if (newMaxComments !== maxComments && newMaxComments !== NaN) {
+    maxComments = parseInt(newMaxComments);
```

Comment:
newMaxComments is already derived from calling parseInt (line 54), is it necessary to call it again?